CONTRIBUTING.md

```diff
@@ -13,27 +13,33 @@ Otherwise, clone the repo and base your changes from the `dev` branch of your re
 
 Try to follow these naming conventions so branches will be organized in a list, easily distinguished, and easily traced to the work involved.
 
-In general, follow the form `prefix/JIRA-###-short-description`
+In general, follow the form `prefix/[dev|gh]-###-short-description`
 
-Prefixes are described below. Include the Jira ID (or Github Issue) for this work (or the primary ID when multiple) if one exists. Then a few words to concisely summarize what it is.
+* `prefix` options are described below.
+* `[DEV|GH]`choose "DEV" if the work follows a Jira ticket, "GH" if it is a Github issue.
+* `###` is the associated Jira ticket or Github issue number (or the primary ID when multiple).
+* `short-description` a few words to concisely summarize what it is.
 
 Prefix|Change Type|Examples
 ------|-----------|--------
 `ftr/`|_New_ stuff. New feature development, or new framework, utility, etc.|`ftr/dev-123-short-description`
 `mod/`|Updates, refactoring, improvements, or modifications to an _existing_ feature/framework/code. <br/>_(If this is happening as part of new feature development, use `ftr/` instead.)_|`mod/dev-456-short-description`
-`bug/`|Bug fixes that are not being hotfixed|`bug/dev-21-the-problem`
+`bug/`|Bug fixes that are not being hotfixed|`bug/gh-21-the-problem`
```

Comment:
If something is a GH issue AND a jira ticket, is there a preference for which one it's noted as?

Reply:
GH is for public contributors. Jira for project Devs